python/ql/src/semmle/python/objects/ObjectAPI.qll

```diff
@@ -14,887 +11,724 @@ private import semmle.python.pointsto.PointsToContext
 private import semmle.python.pointsto.MRO
 private import semmle.python.types.Builtins
 
-/* Use the term `ObjectSource` to refer to DB entity. Either a CFG node
+/*
```

Comment:
Uh oh. It looks like this entire file has been autoformatted (with two space indents, even). Is there any easy way of getting it back to its non-autoformatted state? As-is, it's a bit difficult to see which changes are relevant, and which are not. (If not, just do the four-space autoformatting, and we can fix up any stray formatting issues in this PR.)